src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -77,7 +77,7 @@ object GroupReadsByUmi {
     }
 
     /** Creates/retrieves a ReadEnds object from a SamRecord and stores it in a temporary attribute for later user. */
-    def apply(rec: SamRecord) : ReadInfo = {
+    def apply(rec: SamRecord, ignoreStrand: Boolean = false) : ReadInfo = {
```

Comment:
For "merged" paired end reads, we assume that the strand of the fragment read comes from R1:
1. if we have R1+/R2-, then the merged fragment read is `R1+ => revcomp(R2-)`
2. if we have R2+/R1-, then the merged fragment read is `R1- => revcomp(R2+)`

So when we are _retrieving_ reads to be compared, we should ignore the strand when slurping in templates, but then condition on strand for which read is `/A` and which read is `/B`.